docs/docker-deployment.md

```diff
@@ -1,13 +1,21 @@
-# Docker deployement
+# Docker deployment
 
 This documents the [Docker][] way to deploy uData.
 If you are intending to perform a classic deployment on a server, you should look at the [Installation section](installation.md).
 
-uData and its extensions are packaged as [Docker image][]
-You can go directly to [the dedicated repository][docker-repository] for an up to date docuementation
-and some sample configuration using [Docker Compose][].
+uData and its extensions are packaged and distributed as [Docker images][Docker image].
```

Comment:
The second bracket content is not required if the reference is the same as the link label. Here by the way, it's either you have `[Docker images][Docker images]` or `[Docker images][]`, either you change back the reference at the end of the file (see the next comment)